### ISSUE TITLE
Unable to call appium driver finder method. Locator Strategy 'accessibility id' is not supported for this session

### ISSUE THREAD

**Issue by waisong-anacle** [appium-userland/appium-flutter-driver#387]
I am trying to find element via the appium driver finder method, but this error showed. Does this mean that if i set automationName as flutter then would be unable to call those finder method from appium driver such as findElement()? The reason i need to use appium driver method is because i trying to select datetime from flutter material showDatePicker(), and i don't think there is a way to add key into the calendar and clock view. After checked with appium inspector it seem to have accessibility id to trigger the edit button of datetime, but i not sure how to select the element with method available in appium-flutter-driver.

> Locator Strategy 'accessibility id' is not supported for this session

Example 

```
capabilities.setCapability("automationName", "flutter");
driver = new IOSDriver(service.getUrl(), capabilities);

driver.findElement(new AppiumBy.ByAccessibilityId("Switch to input"));

```

![Screenshot 2022-12-29 at 4 42 40 PM](https://user-images.githubusercontent.com/90165282/209926271-a3fe130d-5e15-4f6b-beda-dbf41c83d956.png)

The calendar widget
<img width="243" alt="Screenshot 2022-12-29 at 4 50 07 PM" src="https://user-images.githubusercontent.com/90165282/209927212-b38829f5-3794-4880-a3c4-ba3676f4b534.png">

**Stack**
appium java client 8.3.0.   
appium (2.0.0-beta.47) + appium-flutter-driver (1.11.0) + iOS emulator.

**Comment by KazuCocoa** [comment]
As https://github.com/appium-userland/appium-flutter-driver#example, flutter elements via flutter_driver by Flutter team can access via finder and APIs listed in https://github.com/appium-userland/appium-flutter-driver#api. You also can access to the accessibility id etc via its `NATIVE_APP` context via UIA2/XCUITest driver as regular appium usage

**Comment by KazuCocoa** [comment]
https://github.com/appium-userland/appium-flutter-driver/tree/main/example has a few examples

**Comment by waisong-anacle** [comment]
> As https://github.com/appium-userland/appium-flutter-driver#example, flutter elements via flutter_driver by Flutter team can access via finder and APIs listed in https://github.com/appium-userland/appium-flutter-driver#api. You also can access to the accessibility id etc via its `NATIVE_APP` context via UIA2/XCUITest driver as regular appium usage

It's worked. Thanks a lot.

**Comment by hananmalka** [comment]
@waisong-anacle - Can you share how it works for you?
What finder did you use to find element by accessibility_id/id?
